variables.tf

```diff
@@ -71,6 +71,30 @@ variable "machine_zone" {
   default     = "us-central1-a"
 }
 
+variable "machine_region" {
+  description = "The region for the machine instance to be deployed in"
+  type        = string
+  default     = "us-central1"
+}
+
+variable "machine_subnetwork" {
+  description = "The instance subnetwork name (subnet name)"
+  type        = string
+  default     = "default"
+}
+
+variable "machine_network" {
```

Comment:
Seems like we don't use `machine_network` anymore. We should remove this variable, wdyt ?

Reply:
Will need to update examples as they all take `machine_network` as input.

Reply:
yeah agreed, its basically depreciated.

I will update examples and remove that variable.

Reply:
I cant see why the build is failing btw.